src/component/addBook/reusable/modal/Modal.css

```diff
@@ -0,0 +1,56 @@
+/* .backdrop {
+    display: flex;
+    flex-direction: column;
+}
+ */
```

Comment:
- [ ] Kindly, remove this commented code in order to improve the repo's cleanliness. 👍

Reply:
Removed.